tests/tasks/test_logger.yml

```diff
@@ -3,7 +3,8 @@
   vars:
     __logging_tag: "testTag{{ __logging_index }}"
     __logging_message: "testMessage{{ __logging_index }}"
-  command: /bin/logger -i -p local6.info -t "{{ __logging_tag }}" "{{ __logging_message }}"
+  command: "/bin/logger -i -p local6.info -t {{ __logging_tag }}
```

Comment:
with command it is better to use `>-` which will generate a single line string with each newline replaced with a single space

Reply:
Not this command, but I have a requirement to convert to something like this.
```
command: my_command long_arg_012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789 another_arg
```
I could not find a good way to separate the long argument > 80 into multiple lines and concatinate them back to one without white spaces in-between if I use `>` or `|`. Only a solution I found was to make it a string and put `\` at the end of the line where a white space is not needed.

Reply:
right - there is no way to do this without using `\`.  That's a good case for leaving the long yaml line length for readability.